### PR TITLE
Fix python docstrings

### PR DESCRIPTION
**Pull request by danymat**

### CODE REVIEW

File: lua/neogen/configurations/python.lua [hunk 1]
```diff
@@ -151,26 +144,24 @@ return {
                             end
                         end
 
+                        vim.pretty_print(res)
```
Comment (biojerm) [inline lua/neogen/configurations/python.lua:147]:
```suggestion
```
Delete this, I think it is causing a message to pop up when i create a docstring

**Review by biojerm — COMMENTED** [review]
Scenario 1 looks good,  thanks for the fix!

Scenario 3 looks correct and could be at an end state.  But it is not exactly what I was thinking.   

I am doubt these are independent sources but [here](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e#file-docstrings-py-L100) and [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) they show that if they argument is typed with a default value, the info should be in the docstring:

eg:
`times (int, optional): <user provider text>. Defaults to 1.` vs `times (int):`

All that being said I don't see anything in the [actual google spec](https://google.github.io/styleguide/pyguide.html#doc-function-args) that mentions needing the word 'optional' or listing out the default values.  So It seems like an implementation choice.  I am happy with the current state, but wanted to point out the difference.

General comment, I am seeing a new message show up when I call the `generate` function.  I think it is debug code (see attached image)
image

I made a code suggestion on the line I think needs to be deleted.

Thanks again!

**Comment by danymat** [comment]
Hello, remove the debug print !
Referring to issue 3, I'm in favor to keep current implementation as it is official specs. Python projects have tendency to customize their annotations depending on the project.
I'm merging this PR as it seems to be resolved. Thank you for your feedback !
Regards

**Comment by biojerm** [comment]
Thanks again for making the updates. Happy holidays!

**Pull request merged**